src/components/CallView/shared/EmptyCallView.vue

```diff
@@ -148,19 +160,27 @@ export default {
 		},
 
 		helper() {
-			if (this.connectionFailed) {
-				return t('spreed', 'Please try to reload the page')
-			}
-			return ''
+			return this.connectionFailed ? t('spreed', 'Please try to reload the page') : ''
 		},
 
 		message() {
-			if (this.connectionFailed === 'consent') {
-				return t('spreed', 'Recording consent is required')
-			}
-
 			if (this.connectionFailed) {
-				return t('spreed', 'Something went wrong')
+				const statusErrors = {
+					0: t('spreed', 'No response from server'),
+					400: t('spreed', 'Recording consent is required'),
+					403: t('spreed', 'This conversation is read-only'),
+					404: t('spreed', 'Conversation not found or not joined'),
+					412: t('spreed', "Lobby is still active and you're not a moderator"),
+				}
+				const statusCode = this.connectionFailed.meta?.statuscode
```

Comment:
Either this or ensure that getter always returns an object
```suggestion
				const statusCode = this.connectionFailed?.meta?.statuscode
```